conf/log4jSTAF.properties

```diff
@@ -4,27 +4,40 @@
 # documenation of org.apache.log4j.PropertyConfigurator.
 
 # Debugging setting
-#log4j.rootLogger=DEBUG,stdout,logfile
+#rootLogger.level=DEBUG,stdout,logfile
 
 # Deployment setting
-log4j.threshhold=OFF
-log4j.rootLogger=WARN
+rootLogger.level=WARN
 
```

Comment:
Add StdoutLogger for rootlooger.
Also use approach like :
1 - Define all appenders
2 - Define loggers
3 - root logger configuration

Reply:
Please check now and let me know